app/common.php

```diff
@@ -14,12 +14,12 @@
     define('DB_NAME', ltrim($dbopts["path"], '/'));
 }
 
-if (file_exists('config.php')) {
-    require 'config.php';
+if (file_exists(__DIR__.DIRECTORY_SEPARATOR.'config.php')) {
+    require __DIR__.DIRECTORY_SEPARATOR.'config.php';
```

Comment:
I see what you did there...  For a minute, I thought I had the `..` in there, and you'd removed it.  Thanks for adding it!